fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/InclusionQuerySegmentAggregator.java

```diff
@@ -133,17 +133,21 @@ protected SqlQueryData buildCountQuery() throws Exception {
         final String METHODNAME = "buildCountQuery";
         log.entering(CLASSNAME, METHODNAME);
 
-        List<Object> allBindVariables = new ArrayList<>();
-        this.addBindVariables(allBindVariables);
-
         StringBuilder queryString = new StringBuilder();
         queryString.append(SELECT_COUNT_ROOT);
         queryString.append(LEFT_PAREN);
         queryString.append(QuerySegmentAggregator.SELECT_ROOT);
         buildFromClause(queryString, resourceType.getSimpleName());
-        allBindVariables.addAll(this.idsObjects);
-        buildWhereClause(queryString, null);
 
+        // An important step here is to add _id and _lastUpdated and then
+        // the regular bind variables. 
+        List<Object> allBindVariables = new ArrayList<>();
+        allBindVariables.addAll(idsObjects);
+        allBindVariables.addAll(lastUpdatedObjects);
+        this.addBindVariables(allBindVariables);
```

Comment:
are the other regular bind variables are added in this.addBindVariables(allBindVariables)?

Reply:
just checked the codes, and found it's

Reply:
that's right.